### ISSUE TITLE
Tab text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52023]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10

Additional Details:
MAS Violated: MAS1.4.3

Repro Steps:
1)Launch VS Code.
2) Click on View menu item and navigate to Problems
3)Open Color contrast analyser and check the ratio of all the non selected tabs.

Actual:
Color contrast ratio is 2.9:1 of all non selected tab items(Output, Terminal, Debug Console) which is less than required.
Same issue has been observed in below themes:
Light
Light+
Quiet Light
Dark+
Kimbie Dark
Monokai
Solarized Dark
Tomorrow Night Blue

Expected:
Color Contrast ratio should be equal to or greater than 4.5:1

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
The user will not be able to see the name of the tab items.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment for reference: 
![a11y vscode problems tabitems colorcontrast](https://user-images.githubusercontent.com/39204909/41465057-1a83d306-70ba-11e8-93e2-0fbf626b0e65.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Fixed by #54881
